environment.yml

```diff
@@ -1,61 +1,76 @@
 name: guide
 channels:
+  # used by openmm, avoids installing cudatoolkit (multiple GBs)
   - jaimergp/label/unsupported-cudatoolkit-shim
   - conda-forge
-  - plotly
   - defaults
 dependencies:
+  # mdanalysis dependencies
   - cython
-  - distributed==2021.1.0
-  - dask==2021.1.0
-  - python-graphviz
-  - pytest
-  - decorator
-  - docutils
-  - graphviz
-  - ipywidgets<8.0.0
-  - ipython
-  - ipygany
-  - jedi
-  - jinja2
-  - jupyter_contrib_nbextensions
   - matplotlib
-  - nbclient
-  - nbconvert
-  - nbformat
-  - nbsphinx
   - networkx
-  - nglview
-  - notebook
   - numpy
+  - openmm>=7.0.0
   - packaging
   - parmed
-  - parso
-  - pip
-  - plotly
-  - pybtex
-  - python=3.9
-  - pyvista
-  - scikit-image
+  - pyedr>=0.7
   - scikit-learn>=1.1.2
-  - sphinx
-  - sphinxcontrib-bibtex
   - threadpoolctl
-  - tabulate
-  - widgetsnbextension
-  - ipycytoscape>=1.3.0
-  - seaborn>=0.12.1
-  - openmm>=7.0.0
+  # - decorator # not used?
+  # building documentation
+  - docutils
   - pandoc
-  - moviepy
-  - ipyvtklink
+  - pybtex
+  - sphinx
+  - sphinxcontrib-bibtex
+  # - jedi # not used?
+  # - jinja2 # not used?
+  # - nbclient # not used?
+  # - parso # not used?
```

Comment:
You're probably good to remove these. For some history, at one point the user guide environment was having trouble building in good time on RTD, so we pinned down some dependencies to help resolution. Also, some of these (e.g. jinja2) had to be pinned to a specific version in the past for issues that have been since resolved.

Reply:
What's RDT? Makes sense re jinja2 and other such creatures ;-) I suspect there are a few more obsolete packages. The systematic way to eliminate these is to create a single PR for each dep, and confirm if the build works or not... but I don't recommend doing that ;-)

Reply:
RTD stands for ReadTheDocs -- it's one of the checks we run for PRs. It now uses mamba, but when the user guide was first started, our builds would occasionally time out waiting for conda to resolve :)

Reply:
Ah, gotcha read the docs. Yeah the resolution time with native conda is just bafflingly slow... turning environments into lock files really helps, have you ever seen that tried in the MDA-verse?

Reply:
It's been discussed in the past (for the core library) but the general take is that lock files aren't really appropriate for libraries.

My personal take is that they have a space for when you need ironclad production releases but in places like CI create a "leave it and forget it" landmine for some future maintainer to deal with. Indeed this is sort of what happened when we pinned nearly every dependency in this repo.

In any case `mamba` has mostly resolved our issues now.